dfsclient/src/test/java/org/hpccsystems/dfs/client/DFSReadWriteTest.java

```diff
@@ -52,9 +52,9 @@
 @FixMethodOrder(MethodSorters.NAME_ASCENDING)
 public class DFSReadWriteTest extends BaseRemoteTest
 {
-    private static final String[] datasets       = { "~benchmark::integer::20kb"};
-    private static final int[]    expectedCounts = { 1250, 6 };
-    
+    private static final String[] datasets       = { "~benchmark::integer::20kb", "~benchmark::all_types::200kb"};
+    private static final int[]    expectedCounts = { 1250, 5600 };
```

Comment:
did you intend to use datasets[1] and expectedCounts[1] in nullWriteTest?
Line 136 references 5600  a couple of times instead of expectedCounts[1] and line 118 hardcodes "~benchmark::all_types::200kb"